mentoring/mentees/ResourcesForMentees.md

```diff
@@ -0,0 +1,139 @@
+
+## Stay Updated with the CCC Newsletter
+
+To stay informed about the latest news, events, and opportunities within the CCC, including updates from the SIGs, subscribe to the CCC newsletter:
+
+- [Subscribe to the CCC Newsletter](https://confidentialcomputing.io/get-involved/newsletter/)
+
+Your participation and contributions are highly valued as we work together to advance the field of confidential computing. Whether you're new to the community or an experienced professional, there's a place for you in the CCC's SIGs.
+
+For more detailed information on each SIG, including how to subscribe to mailing lists and access archived notes, please visit the [CCC Committees Page](https://confidentialcomputing.io/about/committees/).
+
+
+### Joining a SIG Meeting
+
+All Linux Foundation meetings, including those of the CCC SIGs, are open to anyone interested in participating. Whether you're new to confidential computing or an experienced professional, joining a SIG meeting is a great way to learn more about the field, ask questions, and observe the contributor journeys of others. Here's how you can get involved:
+
+1. **Visit the CCC Committees Page**: To find more information about each SIG, including their focus areas and objectives, visit the [CCC Committees Page](https://confidentialcomputing.io/about/committees/).
+2. **Check the Meeting Schedule**: Each SIG page includes details about upcoming meetings, including dates, times, and how to join.
+3. **Participate or Observe**: You're welcome to actively participate in the discussions or simply observe the proceedings to learn more about the work being done within the CCC.
+
+### Everyone Is Welcome
+
+The Linux Foundation and the CCC emphasize inclusivity and openness, encouraging anyone with an interest in confidential computing to join SIG meetings. Whether you have specific questions, seek to learn more about the field, or wish to share your expertise, your participation is valued and welcomed. Joining a SIG within the Confidential Computing Consortium is an excellent opportunity to contribute to the advancement of secure computing technologies and be part of a community dedicated to shaping the future of privacy and security in the digital age.
+
+
+## Skill Development and Education in Areas Relevant to Confidential Computing
+
+### Linux Foundation: Introduction to Linux
+- **Description**: A good understanding of Linux is essential for working in confidential computing environments, as many projects are Linux-based. This course offers a comprehensive introduction.
+- **Link**: [Introduction to Linux](https://www.edx.org/course/introduction-to-linux)
+
+### GitHub Learning Lab
+- **Description**: Engaging with open-source projects is a key part of confidential computing. GitHub Learning Lab offers guides to get started with contributing to open-source projects.
+- **Link**: [GitHub Learning Lab](https://lab.github.com/)
```

Comment:
This URL returns 404.  Is https://skills.github.com what you intended?

Reply:
That's perfect, thank you!

Reply:
still need to update the link

Reply:
Upated the link to https://github.com/apps/github-learning-lab in my commit.